hmrc/sa-liabilities-sandpit-api
# Feature/di 244 write unit tests for instances

This pull request introduces a comprehensive suite of tests for the Actions and related components in the uk.gov.hmrc.saliabilitiessandpitapi project. The tests cover various aspects of functionality including validation actions, configuration handling, and response handling. The focus is on ensuring that our actions and services correctly interact and handle edge cases as expected.


**How to Test**
- Run All Tests: Execute all tests using your build tool (e.g., sbt test) to verify that all scenarios pass as expected
